cmake/FindLibunwind.cmake

```diff
@@ -18,9 +18,10 @@ find_path(LIBUNWIND_INCLUDE_DIR libunwind.h
 
 find_library(LIBUNWIND_LIBRARIES NAMES unwind )
 if(NOT LIBUNWIND_LIBRARIES STREQUAL "LIBUNWIND_LIBRARIES-NOTFOUND")
-  if (CMAKE_COMPILER_IS_GNUCC)
+  if (CMAKE_COMPILER_IS_GNUCC AND NOT DEPENDS)
```

Comment:
That might be better off done in the caller, since it's find macro ?

Reply:
This was actually not needed anymore. Removed.